test/webserv/integration/common.py

```
@@ -0,0 +1,67 @@
+    globals()[length_var] = length
+
+
+REQUEST_DIR = "test/common/request/"
+REQUEST_GET_2XX_DIR = REQUEST_DIR + "get/2xx/"
+REQUEST_GET_4XX_DIR = REQUEST_DIR + "get/4xx/"
+REQUEST_GET_5XX_DIR = REQUEST_DIR + "get/5xx/"
+
+response_header_get_root_200_close = f"HTTP/1.1 200 OK\r\nConnection: close\r\nContent-Length: {root_index_file_length}\r\nContent-Type: text/html\r\nServer: webserv/1.1\r\n\r\n"
+response_header_get_root_200_keep = f"HTTP/1.1 200 OK\r\nConnection: keep-alive\r\nContent-Length: {root_index_file_length}\r\nContent-Type: text/html\r\nServer: webserv/1.1\r\n\r\n"
+response_header_get_sub_200_close = f"HTTP/1.1 200 OK\r\nConnection: close\r\nContent-Length: {sub_index_file_length}\r\nContent-Type: text/html\r\nServer: webserv/1.1\r\n\r\n"
+response_header_400 = f"HTTP/1.1 400 Bad Request\r\nConnection: close\r\nContent-Length: {bad_request_file_400_length}\r\nContent-Type: text/html\r\nServer: webserv/1.1\r\n\r\n"
```

Ruff

Undefined name `bad_request_file_400_length`

(F821)